webprojects/fundraising.md

```diff
@@ -0,0 +1,30 @@
+# Fundraising website: how to localize and test
+
+[donate.mozilla.org](https://donate.mozilla.org) is our platform to collect donations. The main form is for Mozilla Foundation, but it also hosts a separate form for donations to Thunderbird.
+
+This website relies on third parties to handle payment. The localization of the payment system for debit/credit cards is managed by Stripe which we have no control of.
```

Comment:
As a non native speaker, I really find the last part confusing (I know it was suggested). We also agreed on not using "we".

What about  "… is managed directly by Stripe and it’s not possible to add or change translations."? (assuming that's the meaning of "no control").